apps/google-analytics-4/lambda/package.json

```diff
@@ -13,7 +13,9 @@
     "test:debug": "NODE_ENV=test CIRCLE_SHA1=$(git rev-parse --short HEAD) TS_NODE_TRANSPILE_ONLY=1 mocha -- --inspect --inspect-brk",
     "test:watch": "NODE_ENV=test CIRCLE_SHA1=$(git rev-parse --short HEAD) TS_NODE_TRANSPILE_ONLY=1 mocha --watch --watch-files src --watch-files test",
     "deploy:test": "NODE_ENV=test CIRCLE_SHA1=$(git rev-parse --short HEAD) sls deploy --stage test",
-    "deploy": "NODE_ENV=production CIRCLE_SHA1=$(git rev-parse --short HEAD) sls deploy --stage $STAGE"
+    "deploy": "NODE_ENV=production CIRCLE_SHA1=$(git rev-parse --short HEAD) sls deploy --stage $STAGE",
+    "sls-print": "sls print --stage prd",
```

Comment:
does it make a difference if it's `prd` vs `production`?

Reply:
it definitely does, this ultimately gets plugged into the name of the 
`apps/google-analytics-4/lambda/config/serverless-env.<HERE>.yml`

FILW